docs/topics/Gateway.md

```diff
@@ -563,10 +563,38 @@ A user's presence is their current state on a guild. This event is sent when a u
 |-------|------|-------------|
 | user | [user](#DOCS_USER/user-object) object | the user presence is being updated for |
 | roles | array of snowflakes | roles this user is in |
-| game | object | null, or an object containing one key of "name" |
+| game | ?[game](#DOCS_GATEWAY/game-object) object | null, or the user's current activity |
 | guild_id | snowflake | id of the guild |
 | status | string | either "idle", "online" or "offline" |
 
+### Game Object
+
+###### Game Structure
+
+| Field | Type | Description | Present |
+|-------|------|-------------|---------|
+| name | string | the game's name | Always |
+| type | ?integer | see [Game Types](#DOCS_GATEWAY/game-types)  | Sometimes |
+| url | ?string | stream url, is validated when type is 1 | When type is 1 or 2 |
+
+###### Game Types
+
+| ID | Name | Description |
+|----|------|-------------|
+| 0 | Game | "Playing {}" |
+| 1 | Twitch | "Streaming {}" |
+| 2 | YouTube | "Streaming {}" |
```

Comment:
Is `2` actually Youtube? 

If `1` is a generic "Streaming", Does it support anything other than Twitch urls currently?